src/show.jl

```diff
@@ -121,10 +121,10 @@ function show(io::IO, model::PEtabModel)
     print(io, styled"$(header)$(opt)")
 end
 function show(io::IO, prob::PEtabODEProblem)
-    @unpack probinfo, model_info, nparameters_esimtate = prob
+    @unpack probinfo, model_info, nparameters_estimate = prob
     name = model_info.model.name
     nstates = @sprintf("%d", length(unknowns(model_info.model.sys_mutated)))
```

Comment:
With the change here, does PEtabODEProblem still print correctly after being built?

Reply:
no, the same typo is made in several other files:

https://github.com/search?q=repo%3Asebapersson%2FPEtab.jl%20esimtate&type=code

Reply:
Could you do a search to change it in all files (quite straightforward in vscode), the most critical place to fix this is in the `PEtabODEProblem` struct